modules/apps/portal-template/portal-template-impl/src/main/java/com/liferay/portal/template/internal/ClassLoaderResourceParser.java

```diff
@@ -40,9 +41,7 @@
 public class ClassLoaderResourceParser extends URLResourceParser {
 
 	public ClassLoaderResourceParser() {
-		Class<?> clazz = getClass();
-
-		_classLoader = clazz.getClassLoader();
+		_classLoader = PortalClassLoaderUtil.getClassLoader();
```

Comment:
Hi @dantewang @yuhai ,

For the default class loader in this class, I think we need to remove it.

One reason is, to make it consistent with all other parsers, we need to make ClassLoaderResourceParser only process the template id with a CLASS_LOADER_SEPARATOR. If you do not have it, then it can be considered not a class loader related resource.

Another reason is, doing this we expose portal's class loader implicitly, we should only expose portal's class loader when it is necessary. 

Let's confirm if there is anyone depending on this, if currently no one depends on this, let's remove it. 

If anyone depends on this, two ways we can go:1) make the template id passed in contain class loader info 2) add a new template parser in the impl module only dealing with portal's class loader. I prefer 1), cause it will make it consistent with all the usages in modules.